### ISSUE TITLE
feat: update documentation with new Cluster Scope 

### ISSUE THREAD

**Issue by axel7083** [zakkg3/ClusterSecret#110]
The README.md and documentations need to be updated with the just merged https://github.com/zakkg3/ClusterSecret/pull/101

**Comment by toabi** [comment]
The repo also needs a tag to release the app container with the cluster scoped custom resources.

**Comment by toabi** [comment]
Yes, but chart 0.3.0 doesn't include a new container with new code for e.g. https://github.com/zakkg3/ClusterSecret/pull/101/files#diff-cf40ab8971396384a9fe1e031bebc3589986d40033eec1b7af61943cff3b0117 

The last one is still v0.0.10 https://quay.io/repository/clustersecret/clustersecret?tab=tags

**Comment by dougbw** [comment]
Any updates on the documentation and a release build on this?
